README.md

```diff
@@ -1,12 +1,59 @@
-# CarCare
-Junior Design project repo for Team LTT
+# Car Care
+## Introduction
+Car Care collects diagnostic information from a car's OBD port and displays it on the user's smartphone with various graphics. The data can be viewed in real time or at a later date without requiring an active connection to the OBDII adapter. Alerts can be set up to notify the user when data received is lower or higher than a specified value.
```

Comment:
Change `OBD port` to `OBD computer through the OBD port using a wireless Bluetooth adapter`